codemods/jscodeshift/next/13/remove-get-static-props/README.md

```diff
@@ -1,27 +1,26 @@
 # remove-get-static-props
-The `app` directory deprecates the `getStaticProps()` data fetching function in the `pages` directory.
+
+The `getStaticProps` function is no longer available in the `app` directory.
 
 This codemod creates comments indicating where to remove the deprecated `getStaticProps()` functions.
```

Comment:
Should we include that this codemod also creates a boilerplate hooks like `const data = getData()`?